indra/llcommon/llnametable.h

```diff
@@ -55,16 +55,16 @@ class LLNameTable
 		mNameMap[tablename] = data;
 	}
 
-	BOOL checkName(const std::string& name) const
+    bool checkName(const std::string& name) const
```

Comment:
something happened to the indentation here

Reply:
Yeah, a tab became spaces. Tying here to find the git setting that makes this an actual change, because it seems to ignore whitespace changes....

Reply:
Indentation should now be fixed.